packages/autorest.python/autorest/codegen/models/property.py

```diff
@@ -45,7 +45,7 @@ def pylint_disable(self) -> str:
     def description(self, *, is_operation_file: bool) -> str:
         from .model_type import ModelType
 
-        description = self.yaml_data["description"]
+        description = self.yaml_data.get("description", "")
```

Comment:
what are we seeing without description?

Reply:
Adding this since found generator will crash here is property has no description in yaml_data.
This description is used to provide rest_field comment like below:
~~~
class InnerError(_model_base.Model):
    innererror = rest_field(name="innererror")  # Inner error.
~~~

If there is no description, it will be:
~~~
class InnerError(_model_base.Model):
    innererror = rest_field(name="innererror")
~~~

And I see we have code to handle case that there is no description such as https://github.com/Azure/autorest.python/blob/autorestv3/packages/autorest.python/autorest/codegen/serializers/model_serializer.py#L20, I think it's safe to use empty description in case it's not in the yaml_data?